packages/config/src/layer2s/linea.ts

```diff
@@ -298,23 +296,28 @@ export const linea: Layer2 = {
     ),
     discovery.contractAsPermissioned(
       discovery.getContract('Roles'),
-      `Module to the AdminMultisig. Allows to add additional members to the multisig via permissions to call functions specified by roles. ${(() => {
-        const roles = discovery.getContract('Roles')
-        assert(roles.values !== undefined)
-        const rolesCount = Object.entries(
-          roles.values.roles['roles' as keyof ContractValue],
-        ).length
-        assert(rolesCount === 0)
-
-        return 'Currently there are no additional members.'
-      })()}`,
+      `Module to the AdminMultisig. Allows to add additional members to the multisig via permissions to call functions specified by roles.`,
     ),
     {
       accounts: operators,
       name: 'Operators',
       description:
         'The operators are allowed to prove blocks and post the corresponding transaction data.',
     },
+    // FIXME: Add value transformer
+    {
+      accounts: [
+        {
+          address: EthereumAddress(
+            '0x453B3A4b4d64B4E6f472A306c3D4Fc318C34bbA8',
```

Comment:
don't hardcode the address

Reply:
[chore: remove hardcoded zodiac pauser](https://github.com/l2beat/l2beat/pull/2565/commits/565a7714a4939e24e98f52e493a20906fd29632d)